tony-history-server/app/utils/HdfsUtils.java

```diff
@@ -48,13 +61,26 @@ static String contentOfHdfsFile(FileSystem fs, Path filePath) {
     }
   }
 
+  /**
+   * Return all paths that satisfy the <code>fn</code> predicate.
```

Comment:
It doesn't make a difference here, but in general, `{@code ...}` is preferable to using `<code>...</code>` as `{@code ...}` lets you embed `<` and `>` without escaping them as `&lt;` and `&gt;`. Ref: https://stackoverflow.com/a/1671106/1128392

Reply:
I was considering using backticks (\`) to wrap the variable name in the doc, but I think I saw it somewhere where they use `<code></code>` as the variable's wrapper. Which one do you guys prefer? @erwa @oliverhu @hungj

Reply:
Backticks is non-standard and won't cause the enclosed text to get rendered as monospace when Javadoc HTML is generated. Let's stick with `{@code ...}`.

Reply:
Resolved: https://github.com/linkedin/TonY/pull/62/commits/63c80af94299a1db84d287f19e8e2c3ed91738de